### PR TITLE
Make localnet track all shards by default

### PR DESCRIPTION
**Pull request by mzhangmzz**
This makes localnet nodes track all shards by default, otherwise, I often need to manually change the config for localnet nodes.

### CODE REVIEW

File: nearcore/src/config.rs [hunk 1]
```diff
@@ -1074,6 +1074,8 @@ pub fn create_testnet_configs_from_seeds(
         config.rpc.get_or_insert(Default::default()).enable_debug_rpc = true;
         config.consensus.min_block_production_delay = Duration::from_millis(600);
         config.consensus.max_block_production_delay = Duration::from_millis(2000);
+        // This will set all nodes to track all shards
+        config.tracked_shards = vec![0];
```
Comment (matklad) [inline nearcore/src/config.rs:1078]: If this is about localnet, can we rename fn to `create_localnet_configs_from_seeds` as a drive-by?

**Review by matklad — APPROVED** [review]

**Comment by akhi3030** [comment]
I think this is not being actively worked on.
